vaadinfrontend/src/main/java/life/qbic/datamanager/views/layouts/Card.java

```diff
@@ -0,0 +1,45 @@
+package life.qbic.datamanager.views.layouts;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+
+/**
+ * <b>Card, A {@link Composite} component containing a {@link Div} hosting the content to be shown
+ * on a card within a {@link PageComponent} in the data-manager-app</b>
+ */
+public class Card extends Composite<Div> {
+
+  private final Span title = new Span("");
```

Comment:
I would argue a simple card is empty.

Reply:
I thought the idea was to enable the user to add specific content to the card which should be automatically styled?
I'm happy to remove the title from this card, however we would then need to decide on how we deal with small changes to the basic card design (e.g. by introducing multiple components for small changes such as titlecard, cardwithbuttons etc...)